pkg/controlplane/controller.go

```diff
@@ -147,14 +91,7 @@ func (c *Controller) Start() {
 		return
 	}
 
-	// Reconcile during first run removing itself until server is ready.
-	endpointPorts := createEndpointPortSpec(c.PublicServicePort, "https")
-	if err := c.EndpointReconciler.RemoveEndpoints(kubernetesServiceName, c.PublicIP, endpointPorts); err == nil {
-		klog.Error("Found stale data, removed previous endpoints on kubernetes service, apiserver didn't exit successfully previously")
-	} else if !storage.IsNotFound(err) {
-		klog.Errorf("Error removing old endpoints from kubernetes service: %v", err)
-	}
-
+	repairClusterIPs := servicecontroller.NewRepair(c.ServiceClusterIPInterval, c.client.CoreV1(), c.client.EventsV1(), &c.ServiceClusterIPRange, c.ServiceClusterIPRegistry, &c.SecondaryServiceClusterIPRange, c.SecondaryServiceClusterIPRegistry)
```

Comment:
Currently we're using different repair loops based on the FG `MultiCIDRServiceAllocator` - why you're changing this?